src/Compilers/VisualBasic/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.vb

```diff
@@ -5285,7 +5287,162 @@ End Class")
 ")
         End Sub
 
+        <Fact>
+        Public Sub ComplexTypes()
+            Dim originalSourceCode = "
+Imports System
+Imports System.Collections.Generic
 
+Class C
+    Sub G()
+        Dim <N:0>a</N:0> = New With {.Key = ""a"", .Value = New List(Of Tuple(Of Integer, Integer))()}
+        Dim <N:1>b</N:1> = Tuple.Create(5, a)
```

Comment:
Thank you! Added a test and made similar changes to VB. An existing VB test fails.

---
In reply to: [116093622](https://github.com/dotnet/roslyn/pull/19283#discussion_r116093622) [](ancestors = 116093622)